documentation/user-interface/report-view.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+title: Report View
+permalink: /documentation/user-interface/report-view.html
+---
+# Report View
+
+The **Report** view displays a detailed report about a test run. It allows you to view the results in different formats, analyze them, restart failed tests, and export the report to a file.
+
+IMAGE
+
+* [Analyzing Test Results](#analyzing-test-results)
+  * [Working with a Grid](#working-with-a-grid)
+    * [Examining Failed Tests](#examining-failed-tests)
+    * [Sorting Data](#sorting-data)
+    * [Searching for Tests](#searching-for-tests)
+    * [Grouping Data Records](#grouping-data-records)
+* [Rerunning Tests](#rerunning-tests)
+* [Exporting the Report](#exporting-the-report)
+* [Removing the Report](#removing-the-report)
+* [Getting More Report Formats](#getting-more-report-formats)
+
+## Analyzing Test Results
+
+The **Report** view can present data in different formats. They include machine-readable formats like JSON or formats that better fit for human examination like a grid view. To select a format, use a drop-down list in the report header.
+
+IMAGE
+
+TestCafe Studio provides the following report formats out of the box:
+
+* Grid view
+* JSON
+* xUnit
+* List
+* Minimal
+* Spec
+

```

Comment:
May be, we should mention `Getting More Report Formats` here?

Reply:
Probably we should mention about the automatic plugins detection here?